client/app/scripts/liveblog-edit/views/main.html

```diff
@@ -186,8 +186,10 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                 </div>
                 <div class="header--live">
                     <a href="{{ publicUrl }}" target="_blank" ng-disabled="!publicUrl" translate>Live</a>
-                    <button class="btn pull-right" ng-click="filterHighlight()" translate>All</button>
-                    <button class="btn pull-right" ng-click="filterHighlight(true)" translate>Highlights</button>
+                </div>
+                <div class="header--filters">                                        
+                    <button class="btn btn-xs active" ng-click="filterHighlight(); filter.isSelected ? (filter.isSelected = !filter.isSelected) : ''" ng-class="{true: 'active', false: ''}[!filter.isSelected]" tooltip-placement="bottom" tooltip="Show all" translate>All</button>
```

Comment:
Sure, just keep in mind this is like a radio toggle between two stated - either All (default) or Highlights is selected. Thanks.